code/datums/cookingrecipes.dm

```diff
@@ -1106,6 +1114,13 @@ ABSTRACT_TYPE(/datum/cookingrecipe/mixer)
 	output = /obj/item/reagent_containers/food/snacks/cinnamonbun
 	category = "Pastries and bread-likes"
 
+/datum/cookingrecipe/oven/chocolate_cherry
+	item1 = /obj/item/reagent_containers/food/snacks/candy/chocolate
+	item2 = /obj/item/reagent_containers/food/snacks/plant/cherry
+	item3 = /obj/item/reagent_containers/food/snacks/condiment/cream
+	cookbonus = 3
+	output = /obj/item/reagent_containers/food/snacks/chocolate_cherry
+
```

Comment:
no category?

Reply:
I noticed that a lot of existing recipes didn't have a category, so I decided not to give it one. I can give it one thought but it'll probably be in a category of its own